src/app/components/app-essentials-loader/AppEssentialsLoader.tsx

```diff
@@ -72,7 +73,18 @@ class AppEssentialsLoader extends React.Component<Props, State> {
 
         let formData;
         if ( mellomlagring && mellomlagring.formData) {
-            formData = { ... mellomlagring.formData }
+            const attachments: Attachment[] = mellomlagring.formData[AppFormField.legeerklæring].map(elem => {
+                const { file: { name, lastModified, type }, url, pending, uploaded} = elem;
+                const file = new File(['foo'], name, { type, lastModified });
```

Comment:
Se om du får laget eget typescript alternativ for File, slik at du slipper å bruke new File